.github/workflows/flutter.yml

```diff
@@ -4,6 +4,7 @@ on:
     branches:
       - main
       - release/**
+      - feat/**
```

Comment:
will remove before merging, this is so ci runs since we merge into another feat branch